addons/website_forum/views/website_forum.xml

```diff
@@ -142,129 +127,205 @@
-                            <a t-attf-href="/forum/#{ slug(forum) }/flagged_queue"
-                                t-attf-class="nav-link #{flagged_queue_active and 'active' or ''}">Spam</a>
-                        </li>
-                    </ul>
-                    <ul class="navbar-nav ml-auto">
-                        <li t-if="not is_public_user" class="nav-item">
-                            <a t-attf-href="/profile/user/#{uid}?forum_id=#{forum.id}" t-attf-class="text-dark font-weight-bold nav-link#{searches.get('my_profile') and ' active' or ''}">
-                                <i class="fa fa-diamond text-secondary" /> <t t-esc="user_id.karma"/>
+    <div class="navbar navbar-expand-sm navbar-light">
+        <div class="container flex-wrap flex-md-nowrap">
+            <a t-if="forums_count and forums_count > 1" class="btn btn-light border mr-2" href="/forum/" title="All forums">
+                <i class="fa fa-chevron-left mr-1"/>All Forums
+            </a>
+
+            <!-- Desktop -->
+            <ul class="navbar-nav mr-auto d-none d-lg-flex">
+                <li class="nav-item">
+                    <a t-attf-href="/forum/#{ slug(forum) }" t-attf-class="nav-link #{question_count and 'active' or ''}">Topics</a>
+                </li>
+                <li class="nav-item">
+                    <a t-attf-href="/profile/users" t-attf-class="nav-link #{searches.get('users') and 'active' or ''}">People</a>
+                </li>
+                <li class="nav-item">
+                    <a t-attf-href="/forum/#{ slug(forum) }/tag" t-attf-class="nav-link #{searches.get('tags') and 'active' or ''}">Tags</a>
+                </li>
+                <li class="nav-item">
+                    <a t-attf-href="/profile/ranks_badges?badge_category=forum" t-attf-class="nav-link #{searches.get('badges') and 'active' or ''}">Badges</a>
+                </li>
+                <li class="nav-item">
+                    <a t-attf-href="/forum/#{ slug(forum) }/faq" t-attf-class="nav-link #{header.get('is_guidelines') and 'active' or ''}">About</a>
+                </li>
+                <span class="navbar-text" groups="base.group_erp_manager">|</span>
+                <li class="nav-item dropdown mx-auto" groups="base.group_erp_manager">
+                    <a class="nav-link dropdown-toggle" type="button" id="dropdownMenu1" data-toggle="dropdown">
+                        <i class="fa fa-cog" role="img"/> Settings
+                    </a>
+                    <div class="dropdown-menu" role="menu" aria-labelledby="dropdownMenu1">
+                        <a role="menuitem" tabindex="-1" t-attf-href="/forum/#{slug(forum)}/edit_welcome_message?enable_editor=1" class="dropdown-item">Design Welcome Message</a>
+                        <a role="menuitem" tabindex="-1" t-attf-href="/web#id=#{forum.id}&amp;view_type=form&amp;model=forum.forum" class="dropdown-item">Edit Forum in the backend</a>
```

Comment:
@sbuhl  do we really need it ? We should use Edit in backend from the top menu bar instead, after hack the system to allow non website_published record. no?

Reply:
@JKE-be 
This option _edit in backend_  was already there. It was called _Forum Settings_ and was next to the search input. But I do agree, we can remove it.
If we remove it, we only have this _Design Welcome Message_ I think we should display it in Edit mode (https://tinyurl.com/y24y533j) and get rid of this option (https://tinyurl.com/y2hpp5vs).

Reply:
ok, let's go to mode the edit in backend on the top .
And we remove option and controller to edit welcome banner and use

We add editable in condition and class css_non_editable_mode_hidden to let admin edit it

We will do it ourself after merge

Reply:
@JKE-be So we merge this way and adapt later if I understand you well?